webapp/src/components/workspaceSwitcher/workspaceOptions.scss

```diff
@@ -0,0 +1,67 @@
+.workspaceOption {
```

Comment:
If workspaceOption is a component, shouldn't it be `.WorkspaceOption`? and, because `.WorkspaceOption` is alwais inside a `WorkspaceOptions` shouldn't it be inside `WorkspaceOptions` class?

Reply:
To avoid collitions on css when this is running as a plugin we add always the `.focalboard-body` class as prefix, otherwise there are some risk of css classes collition.

Reply:
this `.focalboard-body` applies to most of the classes defined in the focalboard css, so it also appies to the other CSS files submitted here.

Reply:
Have renamed this to just `Options`.